src/constants.js

```diff
@@ -6,5 +6,8 @@ export default {
     copy: {
```

Comment:
@ob6160 By 'All voucher validity' I mean the ones that are checked by the server.